tests/src/Functional/EventIntegrationTest.php

```diff
@@ -132,12 +133,15 @@ public function testCreateEvent(): void {
     $assert_session->responseContains('text-0.txt');
     $assert_session->pageTextContains('Sat, 01/22/2022 - 02:12');
     $assert_session->pageTextContains('Thu, 02/24/2022 - 20:00');
+    $assert_session->pageTextContains('https://europa.eu');
+    $assert_session->elementExists('css', 'div > a[href="https://europa.eu"]');
```

Comment:
Like below: If we use 'classy' theme, we can use more precise selectors and be more confident that the text and href are actually from that specific link.